src/server/on-off-farm/controller.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * The first question to start the journey.
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const onOffFarmGetController = {
+  handler(req, h) {
+    req.yar.set('animal', 'cow')
+    const onOffFarm = req.yar.get('onOffFarm')
+
+    return h.view('on-off-farm/index', {
+      pageTitle: 'Are you moving the cattle on or off your farm?',
+      heading: 'Are you moving the cattle on or off your farm?',
+      onOffFarm: {
+        value: onOffFarm
+      }
+    })
+  }
+}
+
+export const onOffFarmPostController = {
+  handler(req, res) {
+    const { onOffFarm } = req.payload
+
+    if (!onOffFarm) {
+      return res.view('on-off-farm/index', {
+        pageTitle: 'Are you moving the cattle on or off your farm?',
+        heading: 'Are you moving the cattle on or off your farm?',
+        errorMessage: {
+          text: 'Select if you are moving cattle on or off your farm?'
+        }
+      })
+    }
+
+    req.yar.set('onOffFarm', onOffFarm)
```

Comment:
Expecting an await here again, if there's a spill-over to redis state.   Otherwise we're in 'save-and-pray' territory - the user will think (on failure to get to redis) that they've succeeded & continue to the next page, but in reality it'll be missing from their page.

I'm sure there's a reason we're seeing a synchronous interface here, but it isn't sitting right with me!

Reply:
see above comments